src/mscorlib/src/System/Reflection/Emit/CustomAttributeBuilder.cs

```diff
@@ -301,7 +301,7 @@ private static void VerifyTypeAndPassedObjectType(Type type, Type passedType, st
             }
             if (passedType == typeof(IntPtr) || passedType == typeof(UIntPtr))
             {
-                throw new ArgumentException(SR.Argument_BadParameterTypeForCAB, paramName);
+                throw new ArgumentException(SR.Format(SR.Argument_BadParameterTypeForCAB, passedType.FullName), paramName);
```

Comment:
I think calling `ToString()` is better than `FullName` for `Type` objects. `FullName` is intended to roundtrip with `Type.GetType()` so it can very wordy in some situations (generic instances) while returning `null` in others (open types.) `ToString()` is designed for human readability.